e2e/etcdctlv3_test.go

```diff
@@ -211,12 +288,20 @@ func ctlV3Put(cx ctlCtx, key, value string) error {
 	return spawnWithExpect(cmdArgs, "OK")
 }
 
-func ctlV3Get(cx ctlCtx, key, value string) error {
+func ctlV3Get(cx ctlCtx, key string, pairs ...string) error {
```

Comment:
pairs should probably be an array of a data type like `type kv struct { key string; val string; }` instead of having a string array; a flat array of strings is sort of confusing for something called "pairs"